README.md

```diff
@@ -0,0 +1,121 @@
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [portofolio] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[portofolio]** is my personal portofolio.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Tech stacks used on the project are.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">html</a></li>
+    <li><a href="#">css</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    
+  </ul>
+</details>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+### Install
+
+Install this project with:
+npm i
+
+### Usage
+
+To run the project, open index.html
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [@habitoz](https://github.com/habitoz)
+- Linkedin: [@habitoz](https://www.linkedin.com/in/haben-mehari-593531140)
+
```

Comment:
it would be better if u could add the name and profile of the other author (your partner from the admission trial)